src/network/core_utils.rs

```diff
@@ -34,6 +34,99 @@ impl CoreUtils {
         final_slice.join(":")
     }
 
+    //count interfaces on bridge
+    #[tokio::main]
+    #[allow(irrefutable_let_patterns)]
+    pub async fn bridge_count_connected_interfaces(
+        ifname: &str,
+    ) -> Result<Vec<u32>, std::io::Error> {
```

Comment:
Why do we return an vector, it should be enough to return a bool.

Reply:
I just thought in future if need to query connected `veth`'s  then function should support so. So it returns a list of  id's of connected veth's

Reply:
I do not see why we need this and this will prevent us from an early exit when we found one interface.

Reply:
no issues I'll convert this to `bool` if any interface is found.